api/src/test/java/org/openmrs/module/appointments/service/impl/DefaultTeleconsultationAppointmentPatientEmailNotifierTest.java

```diff
@@ -31,6 +32,7 @@
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(Context.class)
+@PowerMockIgnore( {"javax.*", "org.apache.*", "org.slf4j.*"} )
```

Comment:
This is just an incidental fix- this was throwing stack traces in the logs during the build, and adding this ignore line fixes that.